service-front/app/test/CommonTest/Service/Lpa/SiriusLpaFactoryTest.php

```diff
@@ -65,7 +65,16 @@ public function testCanCreateEmptyLpa()
         $this->assertInstanceOf(Lpa::class, $lpa);
         $this->assertEquals('1234', $lpa->getUId());
     }
+    public function testLpaHasSeveranceWarningReturned()
+    {
+        $factory = new Sirius();
+
+        $lpa = $factory->createLpaFromData(['uId' => '1234','hasSeveranceWarning' => true]);
 
+        $this->assertInstanceOf(Lpa::class, $lpa);
+        $this->assertEquals('1234', $lpa->getUId());
+        $this->assertEquals(true, $lpa->getApplicationHasSeveranceWarning());
+    }
```

Comment:
```suggestion
    }
    
```

Whilst you're in this file update all the test methods with a void return as thats good "code scout" behaviour.